src/tests/e2e/api/chef_api_e2e_test.py

```diff
@@ -1,34 +1,58 @@
-from src.constants.http_status_code import HttpStatus
+from src.constants.http import HttpStatus
+from src.constants.oauth2 import Roles
+from src.env_config import get_env_config_instance
 from src.tests.e2e.base_flask_setup_test import BaseFlaskSetupTest
 from src.tests.utils.fixtures.json_entities_fixture import build_chef
 from src.tests.utils.fixtures.mapping_orm_fixtures import create_order_with_procedure
+from src.tests.utils.fixtures.oauth2_fixture import build_client, build_user
+from src.utils.oauth2_util import build_user_credential_access_token
 
 
 class ChefApiE2ETest(BaseFlaskSetupTest):
     def after_base_setup(self):
-        pass
+        env_config = get_env_config_instance()
+        user = build_user(roles=Roles.ADMINISTRATOR.value)
+        client = build_client()
+        scopes = ["READ"]
+        self.access_token = build_user_credential_access_token(
+            user, client, env_config.oauth2_secret_key, scopes
+        )
 
     def test_add_chef_request(self):
         json_chef = build_chef()
-        request = self.client.post("/chefs", json=json_chef)
+        request = self.client.post(
+            "/chefs", query_string={"access_token": self.access_token}, json=json_chef
```

Comment:
this wrong, you don't pass the access token through the query string... you pass it through the autorization header, please use that header instead.

Reply:
Done. Using headers instead.